lib/libc/string/strlen.c

```diff
@@ -126,5 +126,4 @@ strlen(const char *str)
 	}
 
 	/* NOTREACHED */
-	return (0);
```

Comment:
This is not right. Many compilers will warn if you don't end the function like this, even if it is nominally not reached.

Reply:
Not every function in this codebase has this and it’s been compiling fine for years so I don’t know.